lib/tizen_build_target.dart

```diff
@@ -427,7 +427,9 @@ class DotnetTpk {
           (File lib) => lib.copySync(libDir.childFile(lib.basename).path));
     }
 
-    // Keep this value in sync with the latest published nuget version.
+    // TODO(swift-kim): This property is used by projects created before May
+    // 2021. Keep the value up to date until majority of projects are migrated
```

Comment:
How about using a annotation(```@deprecated```)?

Reply:
The annotation is not applicable here so I added some warning message instead. How do you think?
```
The use of PackageReference in Runner.csproj is deprecated. To migrate your project, run:
  rm tizen/Runner.csproj
  flutter-tizen create .
```

Reply:
Oh, I think that's a good idea! Thank you.